README.md

```diff
@@ -28,6 +30,7 @@ The distributed nature of podcasting makes linking to a show on a given platform
 * Podcast Addict: `https://podcastaddict.com/feed/${encodeURIComponent(feedUrl)}`
 * Podcast Guru: `https://app.podcastguru.io/podcast/${appleID}`
 * Podcast Republic: `https://www.podcastrepublic.net/podcast/${appleID}`
+* Podchaser: `https://podchaser.com/f/pod/${appleID}`
```

Comment:
```suggestion
```
Podchaser is already present in the [Resource Links](https://github.com/nathangathright/podcast-platform-links#resource-links) section.